en/docs/fragments/pickup-manager-saml.md

```diff
@@ -2,7 +2,70 @@
 
 ### Register a service provider
 
-insert portal content
+!!! note "Important"
+
+    SAML2 POST Binding requires CORS configs set up. Before configuring the service provider, make sure you add the following configurations to the `<IS_HOME>/repository/conf/deployment.toml` file to allow HTTP POST requests. 
+
+    ```toml
+    [cors]
+    allow_generic_http_requests = true
+    allow_any_origin = false
+    allowed_origins = [
+        "http://localhost:8080"
+    ]
+    allow_subdomains = false
+    supported_methods = [
+        "GET",
+        "POST",
+        "HEAD",
+        "OPTIONS"
+    ]
+    support_any_header = true
+    supported_headers = []
+    exposed_headers = []
+    supports_credentials = true
+    max_age = 3600
+    tag_requests = false
+    ```
+
+The next step is to configure the service provider.
+
+1.  Return to the WSO2 IS management console.
+
+2.  Navigate to **Main**>**Identity**>**Service Providers** and click **Add**.
+
+3.  Enter **saml2-web-app-pickup-manager** in the **Service Provider Name** text box,
+    and click **Register**.
+
+4.  In the **Inbound Authentication Configuration** section, click
+    **Configure** under the **SAML2 Web SSO Configuration** section.
+
+    1.  Now set the configuration as follows:
+
+        1.  **Issuer** : `               saml2-web-app-pickup-manager.com             `
+
+        2.  **Assertion Consumer URL** :
+            `                               http://localhost.com:8080/saml2-web-app-pickup-manager.com/home.jsp`                        
+                 
+            Click Yes, in the message that appears.
+
+    2.  Select the following check-boxes:
+        1.  **Enable Response Signing**
+
+        2.  **Enable Single Logout**
+
+        3.  **Enable Attribute Profile**
+
+        4.  **Include Attributes in the Response Always**  
+        
+        5.  **Enable Signature Validation in Authentication Requests and Logout Requests**
+            
+    
+    !!! tip
+        For more information on the advanced configurations
+        refer, [Configuring SAML2 WEB Single-Sign-On](../../../guides/login/webapp-saml/)
```

Comment:
To configure more advanced configurations, see [SAML Configurations](../../login/saml-app-config-advanced).